launchdarkly-android-client-sdk/build.gradle

```diff
@@ -138,19 +138,6 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
     from javadoc.destinationDir
 }
 
-// Force the Javadoc build to fail if there are any Javadoc warnings. See: https://discuss.gradle.org/t/javadoc-fail-on-warning/18141/3
```

Comment:
Is there a modern version of this? Or not relevant anymore?

Reply:
Discussed in slack.  I was unable to find a solution for this "warnings on autogenerated files" problem that works in Gradle.